concurrent/src/test/java/io/airlift/concurrent/TestLocks.java

```diff
@@ -162,7 +162,9 @@ public void testLockingInterruptiblyCanBeInterrupted()
             fail("ExecutionException was expected");
         }
         catch (ExecutionException expected) {
-            assertTrue(expected.getCause() instanceof InterruptedException);
+            if (!(expected.getCause() instanceof InterruptedException)) {
+                fail("Expected failure caused by interruption", expected);
+            }
```

Comment:
I wasn't preserving failure exception in case where it was other than expected. That would be rogue.